source/MRMesh/MRGcodeProcessor.cpp

```diff
@@ -300,7 +296,7 @@ GcodeProcessor::MoveAction GcodeProcessor::moveArc_( const Vector3f& newPoint, b
         {
             const int pointCount = int( res.action.path.size() );
             res.toolDirection.resize( pointCount );
-            const Vector3f rotationAnglesStep_ = ( *inputRotation_ - rotationAngles_ ) / ( pointCount - 1.f );
+            const Vector3f rotationAnglesStep_ = ( newAngles - rotationAngles_ ) / ( pointCount - 1.f );
```

Comment:
what if `pointCount` == 1

Reply:
impossible. in this logic branch minimum value is 11 (as result previous method)